src/SourceCode.Clay.Threading/SourceCode.Clay.Threading.csproj

```diff
@@ -8,6 +8,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="2.9.1" PrivateAssets="All" />
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0-beta2-18618-05" PrivateAssets="All" />
+    <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.2" />
```

Comment:
Consider multi-targeting netcoreapp2.1/2.2

Reply:
System.Threading.Tasks.Extensions is a "null" package for netcore. You only need to multitarget where no polyfill package exists.